core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -94,6 +94,7 @@ trait RootRawlsApiService extends HttpService {
           .replace("scopeSeparator: \",\"", "scopeSeparator: \" \"")
           .replace("url = \"http://petstore.swagger.io/v2/swagger.json\";",
             "url = '/rawls.yaml';")
+          .replace("jsonEditor: false,", "jsonEditor: false, validatorUrl: null, apisSorter: 'alpha',")
```

Comment:
`apisSorter: 'alpha'` keeps the current sorting behavior. `validatorUrl: null` disables the validator badge, which always fails for all environments where the yaml is not available on the public internet.